src/client/scripts/esm/game/chess/selection.js

```diff
@@ -39,8 +41,27 @@ import config from '../config.js';
  * including the legal moves it has available.
  */
 
+/**
+ * Bugs:
+ * - Fixed I think?
+ * To do:
+ * - Allow dragging the board with a second finger while dragging a piece.
```

Comment:
That's interesting behavior! Could be good. Have you also thought of potentially terminating a drag if a 2nd finger is present? Like, I can imagine you go to zoom out with your fingers and one finger touches down a split second before the other, causing you to drag a piece with that finger and pan with the 2nd finger, instead of zooming out. What are your thoughts?

Reply:
Good idea. Dragging pieces by accident could get annoying.
Is to okay if auto-pan is implemented in a future pull request?